src/midje/data/metaconstant.clj

```diff
@@ -72,17 +72,18 @@
   clojure.lang.Seqable
   (seq [this] (seq storage))  ; I'm unhappy to define this, but it's needed by count/empty.
 
+  clojure.lang.Counted
+  (count [this] (count storage))
```

Comment:
Moved the implementation of `count` to `Counted` because I found it strange that `(count some-meta-constant)` wasn't actually invoking the `(count storage)` logic; making this change addressed that..